.gitignore

```diff
@@ -0,0 +1,2 @@
+# Ignore all files that end with .md
+*.md
```

Comment:
- [ ] Smart move with your style of ignoring `*.md` files, however, it is better to only ignore `test.md` alone as `README.md` could also have been ignored by this rule.

- [ ] It is better to also add `node_modules/` as one of the files/directory to ignore as it is a requirement. Node modules directory may stall git operations and make your Github look messy. It could be accidentally added to your project by various installations.